catalog/ui/catalog-ui-search/webpack/config/dev.js

```diff
@@ -1,24 +1,45 @@
+var path = require('path');
 var webpack = require('webpack');
 var merge = require('webpack-merge');
 
 var base = require('./base');
 
+var resolve = function (place) {
+    return path.resolve(__dirname, '../../', place)
+};
+
 module.exports = merge.smart(base, {
+    devServer: {
+        progress: true,
+        historyApiFallback: true,
+        inline: true,
+        hot: true,
+        contentBase: [resolve('./node_modules/cesium/Build/'), resolve('/src/main/webapp/')],
+        proxy: {
+            '/search/catalog/**': {
+                target: 'https://localhost:8993',
+                secure: false,
+                changeOrigin: true
+            },
+            '/services/**': {
+                target: 'https://localhost:8993',
+                secure: false,
+                changeOrigin: true
+            }
+        }
+    },
     entry: [
-        'webpack-hot-middleware/client?path=/__webpack_hmr',
-        'webpack/hot/only-dev-server',
         'stack-source-map/register',
         'console-polyfill'
     ],
     module: {
-        loaders: [
-            {
-                test: /\.jsx$/,
-                loaders: ['react-hot']
-            }
-        ]
+        loaders: [{
+            test: /\.jsx$/,
+            loaders: ['react-hot']
+        }]
     },
     plugins: [
+        new webpack.NoErrorsPlugin(),
```

Comment:
You should also add `new webpack.NamedModulesPlugin()` so the console writes the module names instead of their ids.